assets/js/script.js

```diff
@@ -0,0 +1,346 @@
+const KeyBoard = {
+
+    elements: {
+        input: null,
+        main: null,
+        keys: [
+            'ё', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace',
+            'Tab', 'Й', 'Ц', 'У', 'К', 'Е', 'Н', 'Г', 'Ш', 'Щ', 'З', 'Х', 'Ъ', '\\', 'Delete', 
+            'CapsLock', 'Ф', 'Ы', 'В', 'А', 'П', 'Р', 'О', 'Л', 'Д', 'Ж', 'Э', 'Enter', 
+            'Shift', '\\', 'Я', 'Ч', 'С', 'М', 'И', 'Т', 'Ь', 'Б', 'Ю', '/', 'ArrowUp', 'Shift',  
+            'Control', 'Meta', 'Alt', ' ', 'Alt', 'Control', 'ArrowLeft', 'ArrowDown', 'ArrowRight'
+        ],
+
+        keyCode: [  
+            192, 49, 50, 51, 52, 53, 54, 55, 56, 57, 48, 189, 187, 8,
+            9, 81, 87, 69, 82, 84, 89, 85, 73, 79, 80, 219, 221, 220, 46,
+            20, 65, 83, 68, 70, 71, 72, 74, 75, 76, 186, 222, 13, 
+            16, 191, 90, 88, 67, 86, 66, 78, 77, 188, 190, 144, 38, 16, 
+            17, 91, 18, 32, 18, 17, 37, 40, 39
+        ],
+        keyElements: []
+    },
+
+    properties: {
+        capsLock: false,
+        shift_pressed: false,  
+        alt_pressed: false        
+    },
+
+    keyLayoutNow: [], 
+
+    keyLayoutRu: [
+        'ё', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace',
+        'Tab', 'Й', 'Ц', 'У', 'К', 'Е', 'Н', 'Г', 'Ш', 'Щ', 'З', 'Х', 'Ъ', '\\', 'DEL', 
+        'Caps Lock', 'Ф', 'Ы', 'В', 'А', 'П', 'Р', 'О', 'Л', 'Д', 'Ж', 'Э', 'ENTER', 
+        'Shift', '\\', 'Я', 'Ч', 'С', 'М', 'И', 'Т', 'Ь', 'Б', 'Ю', '/', '↑', 'Shift',  
+        'Ctrl', 'Win', 'Alt', '', 'Alt', 'Ctrl', '←', '↓', '→'
+    ],
+
+    keyLayoutEn: [
+        '`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace',
+        'Tab', 'Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P', '[', ']', '\\', 'DEL', 
+        'Caps Lock', 'A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', ';', '\'', 'ENTER', 
+        'Shift', '\\', 'Z', 'X', 'C', 'V', 'B', 'N', 'M', '.', ',', '/', '↑', 'Shift',  
+        'Ctrl', 'Win', 'Alt', '', 'Alt', 'Ctrl', '←', '↓', '→'
+    ],
+
+    init() {
+
+        this.elements.input = document.createElement('textarea');
+        document.body.appendChild(this.elements.input);
+        
+        this.elements.main = document.createElement('div');
+        this.elements.main.classList.add('keyboard');
+        document.body.appendChild(this.elements.main);
+
+        this.createKeys();          
+    },
+
+    createKeys() {
+
+        let isEnglish = localStorage.getItem('isEnglish');
+
+        if ( isEnglish === 'true')
```

Comment:
It can be simpler:

KeyBoard.keyLayoutNow = isEnglish === 'true' ? KeyBoard.keyLayoutEn : KeyBoard.keyLayoutRu;